sktime/datasets/_data_io.py

```diff
@@ -347,102 +347,39 @@ def load_from_tsfile(
     ValueError if return_data_type = numpy3d but the data are unequal length series
     ValueError if return_data_type = numpy2d but the data are multivariate and/
     or unequal length series
-
     """
+    if return_data_type is None:
```

Comment:
return_data_type now defaults to "nested_univ", so should this still be an option? User unlikely to pass None as an argument

Reply:
we could deprecate the `None` arg, that would become effective in 0.13.0